tests/test_project.py

```diff
@@ -2002,13 +2019,18 @@ def do_run(self, args, ignored):
     west_tmpdir.chdir()
     cmd('update')
 
-    actual = cmd('test-extension', stderr=subprocess.STDOUT).splitlines()
-    expected = [
+    expected_warns = [
+        'WARNING: ignoring project Kconfiglib extension command "list"; this is a built in command',  # noqa: E501
```

Comment:
Nit: these `noqa` do not seem necessary:
```suggestion
        'WARNING: ignoring project Kconfiglib extension command "list"; '
        'this is a built in command',
```
Python concatenates strings similar to what C does.

Reply:
Updated